pages/index.md

```diff
@@ -66,8 +53,34 @@ a login link so you can create tunnels.
   <input type='submit' class='button'>
 </form>
 
+## Installation
 
-# What's with the name?
+The program is shipped in one binary that acts both as a server and a client.
```

Comment:
binary -> executable. I think that term is less ambiguous and more familiar to windows users.

Reply:
I turned this into "executable file".